distributed/utils_test.py

```diff
@@ -1905,8 +1908,7 @@ def clean(threads=True, instances=True, processes=True):
     with check_thread_leak() if threads else nullcontext():
         with check_process_leak(check=processes):
             with check_instances() if instances else nullcontext():
-                with _reconfigure():
```

Comment:
there are tests that depended on `def test_example(cleanup):` also calling `_reconfigure`

what should these tests do instead?